ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj

```diff
@@ -52,6 +52,10 @@
     </DefineConstants>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="AsyncClipboardService, Version=1.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\AsyncClipboardService.1.5.0\lib\net452\AsyncClipboardService.dll</HintPath>
```

Comment:
Gant told me earlier today that the WPF interface doesn't support async clipboard access.